oioioi/talent/management/commands/talent_camp_init.py

```diff
@@ -67,64 +67,85 @@ def handle(self, *args, **options):
                     )
             # Trial contest & round (it's last to become the default contest)
             contest, _ = Contest.objects.get_or_create(
                 id="p",
                 name="Kontest próbny",
                 controller_name='oioioi.talent.controllers.TalentOpenContestController',
             )
-            Round.objects.get_or_create(
+            Round.objects.update_or_create(
                 contest=contest,
                 name="Runda próbna",
-                start_date=today,
-                results_date=today,
+                defaults={
+                    "start_date": today,
+                    "results_date": today,
+                }                
             )
-            DashboardMessage.objects.get_or_create(
-                contest=Contest.objects.get(id="p"), content=settings.TALENT_DASHBOARD_MESSAGE
+            DashboardMessage.objects.update_or_create(
+                contest=Contest.objects.get(id="p"),
+                defaults={
+                    "content": settings.TALENT_DASHBOARD_MESSAGE,
+                }
             )
             # Supervision groups
             for i in settings.TALENT_SUPERVISED_IDS:
                 group, _ = Group.objects.get_or_create(name=contest_names[i])
                 # Supervisions
                 for r in Round.objects.filter(contest_id=i):
-                    Supervision.objects.get_or_create(
+                    Supervision.objects.update_or_create(
                         group=group,
                         round=r,
-                        start_date=r.start_date,
-                        end_date=r.end_date,
+                        defaults={
+                            "start_date": r.start_date,
+                            "end_date": r.end_date,
+                        }
                     )
             # Phases
             for r in Round.objects.filter(contest_id__in=settings.TALENT_PHASED_IDS):
-                Phase.objects.get_or_create(
+                Phase.objects.update_or_create(
                     round=r,
-                    start_date=r.end_date,
-                    multiplier=settings.TALENT_SCORE1,
+                    defaults={
+                        "start_date": r.end_date,
+                        "multiplier": settings.TALENT_SCORE1,
+                    }
+                    
                 )
-                Phase.objects.get_or_create(
+                Phase.objects.update_or_create(
                     round=r,
-                    start_date=r.start_date \
+                    defaults={
+                        "start_date": r.start_date \
                             - settings.TALENT_CONTEST_START[r.contest_id] \
                             + settings.TALENT_PHASE2_END,
-                    multiplier=settings.TALENT_SCORE2,
+                        "multiplier": settings.TALENT_SCORE2,
+                    }
                 )
```

Comment:
Tu trzeba po czymś więcej niż runda to wybierać (mamy kilka obiektów per-runda). Chyba nie mamy lepszej opcji niż multiplier.
Kiedyś by to można zreworkować na parse'owanie np. tupli par.